### ISSUE TITLE
cuda error

### ISSUE THREAD

**Issue by zhentaoCoding** [bytedance/lightseq#153]
lightseq must use cuda? can not use cpu inference?

RuntimeError: [CUDA][ERROR] /tmp/build-via-sdist-o3lilw0f/lightseq-2.1.2/lightseq/inference/pywrapper/bert.cc.cu(38): no CUDA-capable device is detected

**Comment by godweiyang** [comment]
> lightseq must use cuda? can not use cpu inference?
> 
> RuntimeError: [CUDA][ERROR] /tmp/build-via-sdist-o3lilw0f/lightseq-2.1.2/lightseq/inference/pywrapper/bert.cc.cu(38): no CUDA-capable device is detected

Yes. LightSeq only support GPU training and inference. We will consider supporting CPU inference in the future.
